src/style.css

```diff
@@ -1 +1,77 @@
 /* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 14px;
+  box-sizing: border-box;
+}
+
+.header {
+  max-width: 1200px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  margin: 0 auto;
```

Comment:
do you update you demo-files?
because here I see `margin: 0 auto` rule and it's ok
but see no effects on your [demo-page](https://marianvolytskyi.github.io/layout_moyo-header/)

Reply:
@ManoylovAC  how to update demo-page ?

Reply:
@MarianVolytskyi Yes you can start `npm run deploy` script again